packages/fluentui/accessibility/test/behaviors/treeItemBehavior-test.tsx

```diff
@@ -50,11 +50,18 @@ describe('TreeItemBehavior', () => {
       expect(expectedResult.keyActions.root.performClick.keyCombinations[0].keyCode).toEqual(keyboardKey['Spacebar']);
     });
 
-    test(`click is executed with 'spacebar' or 'enter' key, when tree item is 'selectable' and tree item has subtree`, () => {
+    test(`selection is executed only with 'spacebar', when tree item is 'selectable' and tree item has no subtree`, () => {
       const expectedResult = treeItemBehavior({ selectable: true, hasSubtree: true });
-      expect(expectedResult.keyActions.root.performClick.keyCombinations).toHaveLength(2);
-      expect(expectedResult.keyActions.root.performClick.keyCombinations[0].keyCode).toEqual(keyboardKey['Spacebar']);
-      expect(expectedResult.keyActions.root.performClick.keyCombinations[1].keyCode).toEqual(keyboardKey['Enter']);
+      expect(expectedResult.keyActions.root.performSelection.keyCombinations).toHaveLength(1);
```

Comment:
now looking to it deeper and we even don't need to send "hasSubtree" prop, because particular line is not depending on this prop. Only 'selectable' prop is important. 
So we could omit it and you can delete it as well from name of the test. :)

Reply:
Done